packages/vc-data/src/wallet/v2.ts

```diff
@@ -0,0 +1,50 @@
+import { CreateVCType, createSubjectContext, createContextConfig, createContext } from '../util/v2'
+
+export type CryptographicWallet = {
+  '@type': 'CryptographicWallet'
+  walletAddress: string
+  blockchain?: string
+  network?: string
+  keyType?: string
+}
+
+export type WalletPersonV2 = {
+  '@type': 'WalletPerson'
+  hasCryptographicWallet: CryptographicWallet
```

Comment:
`hasCryptographicWallet` suggests a boolean value. But the type is CryptographicWallet 🤔